### DISCUSSION TITLE
Autofailed while using no mods on ranked map

### DISCUSSION THREAD

**Discussion by nmi5** [ppy/osu#26905] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When playing a ranked map (extra diff of https://osu.ppy.sh/beatmapsets/436968#osu/941020) I was auto failed as soon as the map went into its first break. 
My HP drained to 0 and I was failed instantly despite being full when break time started. I don't know what caused it, and have not been able to reproduce the fail since. I was fc'ing with no mods on the failed try so that potentially could have something to do with it. Unfortunately I was not recording.

### Version

2024.130.2-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14105918/compressed-logs.zip)

**Comment by bdach** [comment]
Do you have a replay saved of this at least?

**Comment by nmi5** [comment]
> Do you have a replay saved of this at least?

Unfortunately not. I'd never reported a bug before so I didn't know what to do and had already closed the game when I asked on the discord. I'm sorry I can't be of more help.

**Comment by bdach** [comment]
Converting to discussion for now since there's not much to go on here.

If you happen to hit this again please save a replay and attach it here.

**Reply by nmi5** [reply]
It happened again, on a different map, I also was running shadowplay at the same time so I managed to get a video of it happening which will also be attached with logs and a replay file. Sorry I didn't have all this last time this happened. It happened on this map in case that helps at all. https://osu.ppy.sh/beatmapsets/601840#osu/1272209
[Logs+Replay+mp4.zip](https://github.com/ppy/osu/files/14337848/Logs%2BReplay%2Bmp4.zip)

**Comment by peppy** [comment, marked as answer]
Probably https://github.com/ppy/osu/issues/26879.

**Reply by nmi5** [reply]
Bug happened again after the update that should have fixed it. 
[logs+replay.zip](https://github.com/ppy/osu/files/14543753/logs%2Breplay.zip)
